decoder/gym.go

```diff
@@ -484,7 +484,7 @@ func saveGymRecord(ctx context.Context, db db.DbDetails, gym *Gym) {
 			"raid_pokemon_cp = :raid_pokemon_cp, "+
 			"raid_is_exclusive = :raid_is_exclusive, "+
 			"cell_id = :cell_id, "+
-			"deleted = :deleted, "+
+			"deleted = false, "+
```

Comment:
Also in pokestop it's handled like that.. that's why we only see deleted gyms, not pokestops. Thought I will do it like that. 

I think it's more safe like that, nobody can set it to true in decoder. If fort is seen in decoder we should never set it to true :)

Reply:
I am just concerned that it will be 'undone' later by some action that means we want to delete gyms in code and perhaps not enough care will be taken to reset the deleted flag in enough places.  At least if we were to do it like this now we'd know that it was correct for future.
But I am happy to follow your judgement since I'll just ping you next time it breaks anyway :-)